dashboard/src/components/SideMenu/SendFeedback.tsx

```diff
@@ -0,0 +1,74 @@
+import { MdOutlineFeedback } from 'react-icons/md';
+import { FormattedMessage } from 'react-intl';
+
+import {
+  AlertDialog,
+  AlertDialogAction,
+  AlertDialogCancel,
+  AlertDialogContent,
+  AlertDialogDescription,
+  AlertDialogFooter,
+  AlertDialogHeader,
+  AlertDialogTitle,
+  AlertDialogTrigger,
+} from '@/components/ui/alert-dialog';
+
+import {
+  FEEDBACK_ISSUE_URL,
+  FEEDBACK_EMAIL_TO,
+} from '@/utils/constants/general';
+
+import { NavigationMenuItem } from '@/components/ui/navigation-menu';
+
+import NavLink from './NavLink';
+
+const get_encoded_url = (): string => encodeURIComponent(window.location.href);
+
+const onGitHubClick = (): void => {
+  window.open(`${FEEDBACK_ISSUE_URL}&body=${get_encoded_url()}`, '_blank');
```

Comment:
Nit: I would add a subject and a more descriptive body, so we can get standartized issues

something like on github

```[Feedback] Feedback Title

# Feedback Description:

## URL used to send feedback:
<url-here>
```


We don't have markdown on e-mail, so it would be something like this:

```subject: [Feedback] Feedback Title

Feedback Description:

URL used to send feedback:
<url-here>
```

Reply:
Yes, this has been on my mind since yesterday, but I still forgot